src/vr/vr-manager.js

```diff
@@ -25,14 +41,12 @@ class VrManager extends EventHandler {
     constructor(app) {
         super();
 
-        this.isSupported = VrManager.isSupported;
-
         this._index = {};
-        this.displays = [];
-        this.display = null; // primary display (usually the first in list)
 
         this._app = app;
 
+        this.isSupported = VrManager.isSupported;
```

Comment:
Why not initialize it over the constructor aswell? Would save this extra line

Reply:
Because ESLint emits an 'use before defined' error if you do that.

Reply:
Oh, thank you, I didn't realize its a static and normal property of the same class at the same time.

Reply:
Yeah, I have no idea why it was written like that originally. The WebVR stuff is deprecated now and could be deleted from the codebase at some point, so I don't want to mess with this code too much.